build-system/tasks/update-packages.js

```diff
@@ -119,15 +119,13 @@ function patchRegisterElement() {
 }
 
 /**
- * Closure Compiler doesn't recognize .mjs extension yet, so copy the file to
- * have a .js file extension.
+ * transformEs6Packages() doesn't work on subdirectories of dist/,
+ * and I can't figure out which option enables it.
```

Comment:
@rsimha I noticed you added `transformEs6Packages()` as part of Babel 7 migration. Is `global: true` problematic now? We still use it in a few places.

Reply:
I documented this in item 7 of https://github.com/ampproject/amphtml/pull/18574#issue-220819628. IIRC, `global: true` causes _all_ packages in `node_modules` to be transformed, which at the time sounded more risky than transforming packages as needed. See [this](https://github.com/babel/babelify#why-arent-files-in-node_modules-being-transformed) for more details.

Reply:
OK, wasn't sure if there was a concrete problem. Going to see if we can avoid the hack in this PR by enabling `global` but ignoring `devDependencies` paths.